app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -15,7 +15,14 @@ module Form526ClaimFastTrackingConcern
 
   EP_MERGE_BASE_CODES = %w[010 110 020 030 040].freeze
   EP_MERGE_SPECIAL_ISSUE = 'EMP'
-  OPEN_STATUSES = ['CLAIM RECEIVED', 'UNDER REVIEW', 'GATHERING OF EVIDENCE', 'REVIEW OF EVIDENCE'].freeze
+  OPEN_STATUSES = [
+    'CLAIM RECEIVED',
+    'UNDER REVIEW',
+    'GATHERING OF EVIDENCE',
+    'REVIEW OF EVIDENCE',
+    'CLAIM_RECEIVED',
+    'INITIAL_REVIEW'
```

Comment:
These are the two claim statuses from Lighthouse Benefits Claims API that provide parity with EVSS.